shipyard_blueprints/snowflake/shipyard_snowflake/cli/execute_sql.py

```diff
@@ -55,6 +55,7 @@ def main():
         sys.exit(e.exit_code)
     else:
         logger.info("Successfully executed query")
+        logger.info("Closing Connection")
```

Comment:
should be combined into 1 log line since no other code is being executed in between. If you want the new line for the styling should use \n